docker-compose.yml

```diff
@@ -1,12 +1,21 @@
 version: '3'
 services:
+
     web:
-        build: ./
-        container_name: svgtranslate_web
+        build:
+            dockerfile: ./.docker/web/Dockerfile
+            context: .
         ports:
-          - ${TOOLFORGE_DOCKER_PORT}:80
+            - ${TOOLFORGE_DOCKER_PORT}:80
```

Comment:
is this needed? Personally I would hardcode the port you want it to run on and let developers override it with `docker-compose.override.yml`